### DISCUSSION TITLE
App not update on Build | Cordova Android

### DISCUSSION THREAD

**Discussion by cmlaplata** [quasarframework/quasar#9485] [CLI - Capacitor / Cordova modes]
Hello! I make changes to the web components and see the result in the Android studio emulator or on the web, but when I do the "quasar buid -m cordova -T android" and install the application on my phone.
The construction of the app is successful, but then the changes are not shown when installed the app on the cell phone. Just show and older version of the app, a verison without the new changes.

I tried to remove android with "cordova platform remove android" and reinstall it but the same thing keeps happening :(

I'dont know what i need to do.

**Comment by cmlaplata** [comment]
Working, i delete manually the "src-cordova/www" folder, so then use "quasar build -m cordova -T android" and automatic make a new "www" folder ands works fine, 4 hours of trying things, i hope help someone

**Reply by metalsadman** [reply]
bumping app version in your package.json also ensures that the built project isn't coming from the cache.
